site2/docs/reference-metrics.md

```diff
@@ -0,0 +1,243 @@
+---
+id: reference-metrics
+title: Pulsar Metrics
+sidebar_label: Pulsar Metrics
+---
+
+<style type="text/css">
```

Comment:
do we display this to users?

Reply:
it is to set the css style for the table. keep it consistent with other reference pages.

Reply:
ok, it belongs to metadata, make sure it will not be displayed to users.